xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -2154,7 +2154,7 @@ bool CDVDDemuxFFmpeg::IsVideoReady()
       st = m_pFormatContext->streams[idx];
       if (st->codecpar->codec_type == AVMEDIA_TYPE_VIDEO)
       {
-        if (st->codecpar->extradata)
+        if (st->start_time != AV_NOPTS_VALUE && st->codecpar->extradata)
```

Comment:
This is for streams with video whereas the PR title only refers to streams with audio only.

Reply:
I removed this extraneous check from the PR as well as the Description text above.